src/components/shared/ReactFlow/FlowCanvas/TaskNode/AnnotationsEditor/ComputeResourcesEditor.tsx

```diff
@@ -1,102 +1,45 @@
 import { useCallback } from "react";
 
+import { BlockStack } from "@/components/ui/layout";
 import { Heading } from "@/components/ui/typography";
-import type {
-  AnnotationConfig,
-  AnnotationOption,
-  Annotations,
-} from "@/types/annotations";
+import type { AnnotationConfig, Annotations } from "@/types/annotations";
 
 import { AnnotationsInput } from "./AnnotationsInput";
 
-const GPUs: AnnotationOption[] = [
-  { value: "NVIDIA-H200", name: "NVIDIA H200" },
-  // { value: "nvidia-a100-80gb", name: "NVIDIA A100 80GB" },
-  // { value: "nvidia-h100-80gb", name: "NVIDIA H100 80GB" },
-  // { value: "nvidia-h100-mega-80gb", name: "NVIDIA H100 80GB MEGA" },
-  // { value: "nvidia-l4", name: "NVIDIA L4" },
-  // { value: "nvidia-tesla-a100", name: "NVIDIA A100 40GB" },
-  // { value: "nvidia-tesla-p4", name: "NVIDIA Tesla P4" },
-  // { value: "nvidia-tesla-t4", name: "NVIDIA T4" },
-  // { value: "nvidia-tesla-v100", name: "NVIDIA V100" },
-];
-
-const CloudProviders: AnnotationOption[] = [
-  { value: "google", name: "Google Cloud" },
-  { value: "nebius", name: "Nebius Cloud" },
-];
-
-export const COMPUTE_RESOURCES: AnnotationConfig[] = [
```

Comment:
The "Showback cost owner" annotation also needs to be converted to the new format (removed from OSS)